src/skills-builder/skills-builder-header/skillsBuilderHeader.scss

```diff
@@ -0,0 +1,8 @@
+.vertical-line {
+  border-left: 7px solid #D23228;
+  transform: rotate(13deg);
+}
+
+.header-title {
```

Comment:
Instead of creating a CSS class for the `header-title`, you can instead use the [color utility classes from Paragon](https://paragon-openedx.netlify.app/foundations/colors).

So for this one, I think it would be `text-warning-300`. The hex values from Figma _should_ all be represented somewhere in the Paragon spec

Reply:
This is the color in the Figma mocks but it doesn't have a utility class in Paragon. I had added this to match the mocks.

Reply:
I believe this hex color does exist in Paragon as `warning-300`. Are you looking at the Open edX theme perhaps?

![Screen Shot 2023-02-06 at 11 59 39 AM](https://user-images.githubusercontent.com/92897870/217035577-dc6d698f-9d71-473f-a2c3-fb1b462d0e5e.png)

Reply:
I _was_ looking at the Open edX theme. I had no idea until today that you could change the theme on the Paragon site :-O. Thanks for this! I'll fix this up.